### DISCUSSION TITLE
Typescript Go to Definition Acting Weird

### DISCUSSION THREAD

**Discussion by mikecfisher** [zed-industries/zed#14339] [Feature Requests]
I have a React typescript project and when I press `g + d` Zed turns into search and shows me the React component I defined but also the type from node_modules/@types/react however if I do g + D it just shows me the type as expected.

I have another typescript project and it just goes to the definition as expected. In VS Code go to definition works as expected for both projects. 

Any idea what could be happening?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
